test/kuttl/tests/glance_disk_formats/00-deploy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      oc kustomize ../../../../config/samples/layout/single | oc apply -n $NAMESPACE -f -
```

Comment:
remove this line (L5), and instead change `config/samples/disk_format/disk_format.yaml` to be like the following:

```
# Inject inject_metadata config
apiVersion: glance.openstack.org/v1beta1
kind: Glance
metadata:
  name: glance
spec:
  serviceUser: glance
  databaseInstance: openstack
  databaseAccount: glance
  keystoneEndpoint: default
  customServiceConfig: |
    [image_format]
    disk_formats=raw,iso
  glanceAPIs:
    default:
      type: single
      replicas: 1
```
This way you can directly deploy what you need with a single command.
Also, given this layout covers a kuttl test, can we move `disk_format` under `layout` folder?